muni_portal/tests/pages/test_index.py

```diff
@@ -9,10 +9,10 @@ def setUp(self):
 
     def test_index(self):
         response = self.client.get(reverse("index"))
-        assert response.status_code == status.HTTP_200_OK
-        assert "This is the homepage for muni_portal" in response.rendered_content
+        self.assertEquals(response.status_code, status.HTTP_200_OK)
+        self.assertIn("This is the homepage for muni_portal", response.rendered_content)
```

Comment:
thanks. In future let's even use https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.SimpleTestCase.assertContains which covers both these cases